cmd/juno/cli/root.go

```diff
@@ -188,18 +187,25 @@ func initConfig() {
 		}
 		viper.SetConfigFile(filepath.Join(config.Dir, "juno.yaml"))
 		err = viper.ReadInConfig()
-		errpkg.CheckFatal(err, "Failed to read in Config after generation.")
+		if err != nil {
+			Logger.With("Error", err).Error("Error reading config file")
+		}
```

Comment:
- Please use Fatal
- Please don't use Error in the log message since the logger with print Error word because of the function call to Error. We want the output to be easily readable